docs/user/customization.md

```diff
@@ -384,6 +384,70 @@ For example:
     worker-edab0895c59dba7a566f4b955d87d964                     3.11.0-744-g5b05d9d3-dirty   2.2.0             137m
     ```
 
+#### Nodes with Custom Kernel Arguments
+
+Custom kernel arguments can be applied to through manifests as an installer operation, and can also be applied as a [MachineConfig][machine-config] as a day 2 operation. The kernel arguments are applied upon boot and will be honored by the Machine-Config-Operator from then on.
+
+Example application of `nosmt` (disable hyperthreading):
```

Comment:
please pick something else, as that's already handled by install-config.

Reply:
Found [karg](https://www.kernel.org/doc/html/v4.14/admin-guide/kernel-parameters.html) `loglevel=7`, maybe we can use it here as it is harmless and could be useful